Event-Emitter/code.js

```diff
@@ -0,0 +1,43 @@
+class EventEmitter {
+    constructor() {
+        this.events = new Map();
```

Comment:
```suggestion
        this.events = {};
```
no need to use constructure.